letsencrypt/client/ddns/ddns.py

```diff
@@ -0,0 +1,85 @@
+"""generic DDNS authenticator (RFC 2136)"""
+
+import logging
+import subprocess
+
+import zope.interface
+
+from letsencrypt.client import challenge_util
+from letsencrypt.client import interfaces
+
+
+class DDNS(object):
+    """DDNS authenticator.
+
```

Comment:
i like to separate stdlib, dependencies, own code imports. and also keep them in that order.